mirdata/beatles.py

```diff
@@ -4,14 +4,13 @@
 import numpy as np
 import os
 import csv
+import json
 
-from .utils import (get_save_path, get_local_path, validator, BeatData,
-                    ChordData, SectionData, KeyData, load_json_index,
-                    download_from_remote, RemoteFileMetadata, untar)
+import mirdata.utils as utils
```

Comment:
Was this the `pytest` issue? This is probably the better way to do these imports anyway

Reply:
Yes it was. Also, if we want a better reason why, I talked to a friend at google and he said their guidelines is to always import at the module level.